pyproject.toml

```diff
@@ -11,8 +11,7 @@ requires = [
   # we forbid it until we can properly test against it
   "Cython>=0.26.1,<3.0; python_version=='3.6'",
   "Cython>=0.29.21,<3.0; python_version>='3.7'",
-  "numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'",
-  "numpy>=1.19.2; python_version>='3.7' and platform_system!='AIX'",
+  "oldest-supported-numpy",
```

Comment:
btw the "AIX issue" comment above should be dropped at the same time

Reply:
done (forced push with commit adding `oldest-supported-numpy`)

Reply:
thank you !